js/apps/system/_admin/aardvark/APP/frontend/js/views/indicesView.js

```diff
@@ -47,6 +71,24 @@
       });
     },
 
+    rerender: function () {
+      var self = this;
+
+      $.ajax({
+        type: 'GET',
+        cache: false,
+        url: arangoHelper.databaseUrl('/_api/engine'),
```

Comment:
What is the purpose of calling `/_api/engine` repeatedly?

Reply:
Thanks. Will fix.